Client/Components/DashBoard.razor.cs

```diff
@@ -55,5 +57,13 @@ private static string FormatLabel(string label)
         {
             return $"{label.Replace(" ", "-")}-statistic";
         }
+
+        private void HandleFilterChange(IEnumerable<string> selectedValues)
+        {
+            dataMetrics = selectedValues.ToList();
```

Comment:
Clicking the "X" in the filter component causes selectedValues 
to be null and throwing a ArgumentNullException.
![Skärmbild 2024-04-24 100922](https://github.com/filipkauffeldt/DAT257-group-8/assets/77638480/5f84e59c-bac2-4705-8989-32a42900cbff)

Reply:
I removed the ability to do this. The user can still clear their selected parameters by clicking this:
![image](https://github.com/filipkauffeldt/DAT257-group-8/assets/112847802/4e75e030-3290-4fdb-aed2-7341ec400339)

Reply:
Maybe add a label for the empty checkbox?
Something is like "Toggle all" for example

Reply:
Looks great!